apps/trading/pages/portfolio/index.page.tsx

```diff
@@ -1,39 +1,79 @@
+import { Web3Container } from '../../components/web3-container';
 import { t } from '@vegaprotocol/react-helpers';
+import { PositionsContainer } from '@vegaprotocol/positions';
+import { OrderListContainer } from '@vegaprotocol/order-list';
+import { AccountsContainer } from '@vegaprotocol/accounts';
 import { AnchorButton } from '@vegaprotocol/ui-toolkit';
 
+import { WithdrawalsContainer } from './withdrawals/withdrawals-container';
+import { GridTab, GridTabs } from '../../components/grid-tabs';
+
 const Portfolio = () => {
+  const tabClassName = 'p-[16px] pl-[316px]';
+
   return (
-    <div className="p-24">
-      <h1 className="text-h3 mb-12">{t('Portfolio')}</h1>
-      <div className="flex gap-4 mb-12">
-        <AnchorButton data-testid="deposit" href="/portfolio/deposit">
-          {t('Deposit')}
-        </AnchorButton>
-        <AnchorButton
-          data-testid="deposit-tEuro"
-          href="/portfolio/deposit?assetId=8b52d4a3a4b0ffe733cddbc2b67be273816cfeb6ca4c8b339bac03ffba08e4e4"
-        >
-          {t('Deposit tEURO')}
-        </AnchorButton>
-      </div>
-      <div className="flex gap-4">
-        <AnchorButton
-          data-testid="view-withdrawals"
-          href="/portfolio/withdrawals"
-        >
-          {t('View Withdrawals')}
-        </AnchorButton>
-        <AnchorButton data-testid="withdraw" href="/portfolio/withdraw">
-          {t('Withdraw')}
-        </AnchorButton>
-        <AnchorButton
-          data-testid="withdraw-tEuro"
-          href="/portfolio/withdraw?assetId=8b52d4a3a4b0ffe733cddbc2b67be273816cfeb6ca4c8b339bac03ffba08e4e4"
-        >
-          {t('Withdraw tEURO')}
-        </AnchorButton>
-      </div>
-    </div>
+    <Web3Container
```

Comment:
question (blocking): Any reason for wrapping with the Web3Container component here. Should only be needed for the deposit and withdraw/withdrawals pages respectivly?

Reply:
if i remember correctly, the provider is needed for the withdrawals table which is the bottom sections's last tab of the bottom section. the table looks quite different from the design, because it's just whatever we had for the withdrawals page for now, unmodified.

<img width="725" alt="Screenshot 2022-06-01 at 08 49 01" src="https://user-images.githubusercontent.com/105208209/171354820-72e9149f-8b6c-4511-a21e-0d3fa5d5a14b.png">